PWGCF/FemtoUnited/TableProducer/femtodreamConverter.cxx

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019-2024 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file femtodreamConverter.cxx
+/// \brief converter task for femtoDream to femtoUnited
+/// \author Anton Riedel, TU München, anton.riedel@tum.de
+
+#include "PWGCF/DataModel/FemtoDerived.h"
+#include "PWGCF/FemtoUnited/DataModel/FemtoCollisionsDerived.h"
+#include "PWGCF/FemtoUnited/DataModel/FemtoTracksDerived.h"
+#include "PWGCF/FemtoUnited/DataModel/FemtoV0sDerived.h"
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+
+using namespace o2;
+using namespace o2::aod;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct FemtodreamConverter {
+
+  // femto collisions
+  Produces<FUCols> outputCollisions;
+
+  // femto tracks
+  Produces<FUTracks> outputTracks;
+  Produces<FUTrackMasks> outputTrackMasks;
+  // Produces<FTrackDCAs> outputTrackDCAs;
+  // Produces<FTrackExtras> outputTrackExtras;
```

Comment:
If this is going to be enabled at some point, it is OK
Remember not leaving code commented